src/python/datastore/data_handler.py

```diff
@@ -269,6 +269,26 @@ def get_reproduction_help_url(testcase, config):
       testcase.job_type, 'HELP_URL', default=config.reproduction_help_url)
 
 
+def get_formatted_reproduction_help(testcase):
+  """Return url to reproduce the bug."""
+  help_format = get_value_from_job_definition(testcase.job_type, 'HELP_FORMAT')
```

Comment:
instead of HELP_FORMAT, we should use HELP_TEMPLATE (need to add some 1-2 line documentation of the feature in jobs documentation).
also, we should default to some repro_help_template config table attribute ? 

also, i think data_handler has some use of HELP_URL, can you fix it to use this.

Reply:
Renamed and fixed other usage.